src/main/webapp/resources/js/pages/UserGroupsPage/index.js

```diff
@@ -43,8 +43,8 @@ export function UserGroups() {
     >
       <RolesProvider rolesFn={getUserGroupRoles}>
         <Router style={{ height: "100%" }}>
-          <UserGroupsPage path={setBaseUrl("/groups")} />
-          <UserGroupsDetailsPage path={setBaseUrl("/groups/:id")} />
+          <UserGroupsPage baseUrl={"/groups"} path={setBaseUrl("/groups")} />
```

Comment:
Call the `setBaseUrl` on it here. Even better would be creating a constant for the url since it is the same as what you are passing in the `path`

Reply:
Awesome, can you just update the JSDoc for the baseUrl to state that it should already be set.  Easier to understand if we use this component in different places.